### PR TITLE
Added detection and installation of missing packets

### PR DESCRIPTION
**Pull request by JanPrusinowski**
Added detection and installation of packets: 'lspci', 'lsusb' or 'fwts' where it was missing.
Fixes: [https://github.com/Dasharo/open-source-firmware-validation/issues/195](https://github.com/Dasharo/open-source-firmware-validation/issues/195)

### CODE REVIEW

File: dasharo-security/wifi-bluetooth-switch.robot [hunk 1]
```diff
@@ -54,6 +56,8 @@ WBS002.001 Wifi and Bluetooth card power switch enabled (Ubuntu 22.04)
     Set Option State    ${security_menu}    Enable Wi-Fi + BT radios    ${TRUE}
     Save Changes And Reset
     Login To Linux
+    Detect Or Install Package    pciutils
```
Comment (macpijan) [inline dasharo-security/wifi-bluetooth-switch.robot:59]: @JanPrusinowski Please inspect the implemenation of this keyword. I am afraid you need to be root, so first you need to use `Switch To Root User` as well.

File: platform-configs/include/protectli-v1x10.robot [hunk 2]
```diff
@@ -27,6 +27,7 @@ ${DMIDECODE_TYPE}=                  Desktop
 ${NVME_DISK_SUPPORT}=               ${TRUE}
 ${MEASURED_BOOT_SUPPORT}=           ${TRUE}
 
+
```
Comment (macpijan) [inline platform-configs/include/protectli-v1x10.robot:30]: Why adding this empty line?

Comment (JanPrusinowski) [reply platform-configs/include/protectli-v1x10.robot:30]: Precomit modified this file

File: dasharo-security/wifi-bluetooth-switch.robot (hunk 1 above)
Comment (JanPrusinowski) [reply dasharo-security/wifi-bluetooth-switch.robot:59]: Added where it was required

**Review by macpijan — CHANGES_REQUESTED** [review]
Please fix commit name as request here: https://github.com/Dasharo/open-source-firmware-validation/pull/300#pullrequestreview-2092925417

Also:
- [x] these 2 commits can be squashed, no need to merge them separately
- [ ] add `Fixes: #xxxx` in the PR comment to link to the issue you are fixing

**Review by macpijan — APPROVED** [review]
Changes look good, but we have the same problem with sonoff-rest-api as discussed here: https://github.com/Dasharo/open-source-firmware-validation/pull/300#issuecomment-2148340696

This submodule should not be touched here.

**Pull request merged**